src/uwtools/config/formats/yaml.py

```diff
@@ -94,8 +100,9 @@ def _yaml_loader(self) -> type[yaml.SafeLoader]:
         """
         loader = yaml.SafeLoader
         loader.add_constructor(INCLUDE_TAG, self._yaml_include)
-        for tag in TaggedString.TAGS:
-            loader.add_constructor(tag, TaggedString)
+        for tag_class in (UWYAMLConvert, UWYAMLRemove):
+            for tag in getattr(tag_class, "TAGS"):
+                loader.add_constructor(tag, tag_class)
```

Comment:
Add constructors for both the existing `UWYAMLConvert` type (renamed from `TaggedString`), and the new `UWYAMLRemove`.